packages/form-control/src/index.ts

```diff
@@ -0,0 +1,4 @@
+export * from './FormControlMixin';
```

Comment:
We should lean toward including file extensions beyond the use of package entry points. In that way we could _possibly_ update to `typescript@4.5` so we have export map support and list these entries there, as an alternative.

Reply:
Separately, do we want to prefer the default entry point a "everything in the package so get ready with your tree shake enabled bundler" or is there an "absolute minimum export" that would be useful to use and allow consumers to build up from?

Reply:
The mixin _could_ be the the base import. I'll admit to not being super familiar with this space (re: TypeScript 4.5 entry points), happy to defer do whatever makes the most sense (any good examples or tutorials on this I should check out to get caught up?).

It could be that eventually we have more options, but given the current state, the mixin is probably the minimum export.